pkg/graph/schema.graphql

```diff
@@ -1357,13 +1357,30 @@ type OperationalSolutionSubtask {
   modifiedDts: Time
 }
 
+type ChangedFields {
+  changes: [Field!]!
+}
+
+type Field {
+  name: String!
+  value: ChangedFieldValue!
+}
+
+union ChangedFieldValue = ChangedFields | FieldValue
+
+type FieldValue {
+  new: String

```

Comment:
What do you think about making `new` and `old` interface types? Not all values are strings.